python/src/scipp/compat/mantid.py

```diff
@@ -111,44 +111,53 @@ def set_bin_masks(bin_masks, dim, index, masked_bins):
         bin_masks[sc.Dim.Spectrum, index][dim, masked_bin].value = True
 
 
+def validate_and_get_unit(unit):
+    known_units = {
+        "DeltaE": [sc.Dim.EnergyTransfer, sc.units.meV],
+        "TOF": [sc.Dim.Tof, sc.units.us],
+        "Wavelength": [sc.Dim.Wavelength, sc.units.angstrom],
+        "Energy": [sc.Dim.Energy, sc.units.meV],
+        "dSpacing": [sc.Dim.DSpacing, sc.units.angstrom],
+        "MomentumTransfer": [sc.Dim.Q, sc.units.dimensionless / sc.units.angstrom],
+        "QSquared": [sc.Dim.Q, sc.units.dimensionless / (sc.units.angstrom * sc.units.angstrom)],
```

Comment:
This should not be the same dimension label, I guess?

Reply:
I did wonder about this and couldn't really come to a conclusion. Is the unit handling not sufficient to differentiate between the two?

Reply:
For this purpose yes, but consider, e.g., the interface of `scipp.neutron.convert`, which uses dimension labels for source and target dimension. `sc.neutron.convert(data, Dim.Q, DimQSquared)`.